test/Connector.t.sol

```diff
@@ -180,71 +182,290 @@ contract ConnectorTest is Test {
         MemberlistLike memberlist = MemberlistLike(token.memberlist());
         uint totalSupplyBefore = token.totalSupply();
 
-        homeConnector.deposit(poolId, trancheId, user, amount);
+        homeConnector.transfer(poolId, trancheId, user, amount);
 
         assertEq(memberlist.members(user), validUntil);
         assertTrue(token.hasMember(user));
         assertEq(token.balanceOf(user), amount);
         assertEq(token.totalSupply(), safeAdd(totalSupplyBefore, amount));
      }
 
-    // function testDepositForNonExistentPoolFails(uint64 poolId) public { }
-    // function testDepositForNonExistentTrancheFails(uint64 poolId) public { }
-    // function testDepositWithoutAllowanceFails(uint64 poolId) public { }
-    // function testDepositFromOtherOriginFails(uint64 poolId) public { }
+    function testTransferForNonExistentPoolFails(uint64 poolId, bytes16 trancheId, address user, uint256 amount, uint256 validUntil) public { 
+        vm.assume(validUntil > safeAdd(block.timestamp, minimumDelay));
+
+        // do not add pool
+        vm.expectRevert(bytes("CentrifugeConnector/invalid-pool"));
+        homeConnector.addTranche(poolId, trancheId, "Some Name", "SYMBOL");
+        
+        vm.expectRevert(bytes("CentrifugeConnector/invalid-pool-or-tranche"));
+        homeConnector.updateMember(poolId, trancheId, user, validUntil);
```

Comment:
Is this run after the first revert in this test? I assumed the test ends after the first revert.

Reply:
the test continues and passes in case all the reverts were expected